src/execution_engine.rs

```diff
@@ -50,6 +50,12 @@ impl ExecutionEngine {
         }
     }
 
+    pub fn add_global_mapping(&mut self, global: &AnyValue, addr: *const ()) {
```

Comment:
I think it might make sense to remove this for now since it's not used and is likely to change anyway. Long term it may end up looking something like this:
```rust
struct MemoryAddress(usize);

pub fn add_global_mapping(&(mut?) self, global: &GlobalValue, addr: &MemoryAddress) {}
```

For example, the execution engine outputs an address which can easily be moved or copied out of the EE's scope, so I think that will end up returning a non-Copy `&MemoryAddress` so that the underlying value is kept in scope but is still transmutable to a function.

What do you think?